File: ELibrary.Standard.VB/SplashCoordinator/IProcessCoordinator.vb

Namespace SplashCoordinator

    Public Interface IProcessCoordinator
        ''' <summary>
        ''' Runs when process coordinator has completely processed all processes passed in as parameters
        ''' </summary>
        ''' <remarks></remarks>
        Sub onExitAction()

        ''' <summary>
        ''' It tries to invoke this if user terminates the class before completion
        ''' </summary>
        ''' <remarks></remarks>
        Sub onTerminateAction()


    End Interface

End Namespace

File: EPRO.Library.v3.5/EPRO.Library.v3.5/SplashCoordinator/IProcessCoordinator.cs

namespace ELibrary.Standard.SplashCoordinator
{
    public interface IProcessCoordinator
    {
        /// <summary>
        /// Runs when process coordinator has completely processed all processes passed in as parameters
        /// </summary>
        /// <remarks></remarks>
        void onExitAction();

        /// <summary>
        /// It tries to invoke this if user terminates the class before completion
        /// </summary>
        /// <remarks></remarks>
        void onTerminateAction();
    }
}